_extras/data.md

```diff
@@ -0,0 +1,75 @@
+---
+layout: page
+title: "Workshop data"
+permalink: /data/
+---
+
+## NEON Spatio-temporal Teaching Data Subset
+
+Available on FigShare: 
+[NEON Spatio-temporal Teaching Data Subset](https://figshare.com/articles/Spatio_temporal_Series_Teaching_Data_Subsets/2009586).
+
+**CITATION:** Data Skills Teaching Data Subsets, NEON; Wasser, Leah; Jones, Megan A. (2016): 
+NEON Spatio-temporal Teaching Data Subset. figshare. Fileset. https://doi.org/10.6084/m9.figshare.2009586.v10 
+
+The data and lessons in this workshop were originally developed through a hackathon supported by the 
+[National Ecological Observatory Network (NEON)](https://www.neonscience.org/) - an NSF funded observatory in Boulder, Colorado. 
```

Comment:
@lwasser - There were also other partners but I don't know if that included financial support.  Do they need to be mentioned?

Reply:
YES! how about this @ErinBecker 
`
The data and lessons in this workshop were originally developed through a hackathon funded by the 
+[National Ecological Observatory Network (NEON)](https://www.neonscience.org/) - an NSF funded observatory in Boulder, Colorado. In collaboration with Data Carpentry, SESYNC and CYVERSE
`

the only funding support was NEON! We collaborated with DC. 
Guitly suspects in collaboration:
Data Carpentry (the one and only @tracykteal )
SESYNC (Mike Smorul) 
 Cyverse (Jason Williams) 

@ErinBecker how do you normally handle this? is it worth mentioning these people who were at the core organizing it (these people being jason and mike)? They came out fo the hackathon and helped me organize it. I pulled the data together. Megan and I ran the hackathon. Megan and i then spent like a year working on the lessons after. full backstory :)